content/zh/blog/_posts/2015-03-00-Weekly-Kubernetes-Community-Hangout.md

```diff
@@ -80,7 +79,7 @@ Notes from meeting:
     * Can also with careful flag crafting run rsync over this or set up sshd inside container.
 
     * Some feedback via chat:
-    
+
```

Comment:
Did you mean to include this whitespace change @gavinfish?

Reply:
Seems that my editor automatically deletes the extra `tab` in the line.